src/Apps/Artwork/Components/TrustSignals/AuthenticityCertificate.tsx

```diff
@@ -0,0 +1,42 @@
+import { CertificateIcon } from "@artsy/palette"
+import { AuthenticityCertificate_artwork } from "__generated__/AuthenticityCertificate_artwork.graphql"
+import React from "react"
+import { createFragmentContainer } from "react-relay"
+import { graphql } from "react-relay"
+import { TrustSignal, TrustSignalProps } from "./TrustSignal"
+
+interface AuthenticityCertificateProps
+  extends Omit<TrustSignalProps, "Icon" | "label" | "description"> {
+  artwork: AuthenticityCertificate_artwork
+}
+
+export const AuthenticityCertificate: React.FC<
+  AuthenticityCertificateProps
+> = ({ artwork, ...other }) => {
+  return (
+    artwork.certificateOfAuthenticity &&
```

Comment:
@sweir27 do you know if I need to check for the content of `certificateOfAuthenticity` here? On CMS it is only a checkbox.

Reply:
Good question!

This field takes the form:
```
certificateOfAuthenticity {
  details
}
```

And can have 3 different responses:
```
certificateOfAuthenticity: null
certificateOfAuthenticity: {
  details: "Included"
}
certificateOfAuthenticity: {
  details: "Not included"
}
```

In this case, I believe we _only_ want to show this trust signal if `details === "Included"`. (here's an example artwork: https://staging.artsy.net/artwork/banksy-sale-ends-v-dot-2)

Reply:
Found it: https://github.com/artsy/metaphysics/blob/08ed13430fdd636af6c536f157432ec721ed16b7/src/schema/v1/artwork/index.ts#L922 I need to check. Chatted with @zephraph decided to add a `boolean` field to MP like `hasCertificateOfAuthenticity` to be able to check this rather than comparing the value of current `certificateOfAuthenticity` which has the risk of the value being updated in MP breaking this flag.

Reply:
Yeah, more on that...

Generally it feels like we should _avoid_ cases where we have to string compare on clients for these sorts of features. For limited states we should either lean on booleans or enums from GraphQL's side to ensure that clients have as little redundant behavior as possible.

Reply:
👍 Makes a lot of sense to add to metaphysics!